scripts/service/timeless_pg.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
```

Comment:
@marceloamadeu Why did you not remove puzzle?

Reply:
@vladarefiev Puzzle removed.